_drafts/oryx/orchestration/app-plugins.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Application plugins
+description: Plugins of the Oryx Application
+template: concept-topic-template
+---
+
+# Application plugins
+
+When you create an Oryx Application with the `appBuilder()` function, it creates an instance of `App`. The `App` is a a shell that can be enhanced with (custom) plugins (`AppPlugin`). Plugins allow you to extend Oryx core behaviour without modifying the core code of the framework.
+
+There are several built-in plugins provided by the Oryx framework:
+
+- [`ComponentsPlugin`](#components-plugin)
+- [`InjectionPlugin`](#injection-plugin)
+- [`ResourcePlugin`](#resource-plugin)
+- [`ThemePlugin`](#theme-plugin)
+
+For ordinary application development, there's no need to develop custom plugins.
+
+## Built-in plugins
+
+### Components plugin
+
+The `ComponentsPlugin` is an important plugin that orchestrates the (lazy) loading of components. The plugin registers all the component definitions and loads the implementation whenever required in the DOM.
+
+The plugin exposes the following API:
+
+- `registerComponents()` – registers components
+
+For example, to register more component:
+
+```ts
+import { ComponentsPlugin } from '@spryker-oryx/core';
+
+app.requirePlugin(ComponentsPlugin).registerComponents([
+  ... // <-- Component definitions go here
+])
+```
+
+### Injection plugin
+
+The `InjectionPlugin` plugin manages the dependency injection system (DI). DI is an important concept in Oryx that allows to customise application logic which it nested deep inside the application logic.
```

Comment:
Please rephrase the second sentence. Something is amiss

Reply:
I think there is just a typo `it` => `is`:
```suggestion
The `InjectionPlugin` plugin manages the dependency injection system (DI). DI is an important concept in Oryx that allows to customise application logic which is nested deep inside the application logic.
```
Does it make sense now?

Reply:
I think the double usage of "application logic" is intentional because DI allows you to customize app logic code without rewriting the app logic.